docs/.vuepress/public/cfg/service_desk.yml

```diff
@@ -1227,6 +1227,7 @@ request_vpn_link:
   fields: [
     { label: "Username", key: "username", pattern: '^[\-a-z0-9\._]{3,}$', hint: "Username should include only lowercase letters a-z, 0-9, dash, underscore, or dot.", field: "textfield" },
     { label: "Access IP4 address", key: "IP4_address", pattern: '^\s*((?!169\.254\.|10\.|0\.|127\.|255\.255\.255\.255|192\.168\.)(\d{1,2}|1\d\d|2[0-4]\d|25[0-5])\.)((\d{1,2}|1\d\d|2[0-4]\d|25[0-5])\.){2}(\d{1,2}|1\d\d|2[0-4]\d|25[0-5])\s*$', hint: "Get your IP address here: http://ip4.me", field: "textfield" },
+    {label: "Your public IP address", key: "ipaddress" },
```

Comment:
is this a duplicate or should it be a replacement?

Reply:
doesn't seem to be part of email template https://github.com/huntdatacenter/docs/blob/399d5237dd6d87bc1f8d5190d2384e9ef87537c7/docs/.vuepress/public/cfg/service_desk.yml#L1207-L1225